node/src/reactor.rs

```diff
@@ -674,7 +677,6 @@ where
 
     /// Runs the reactor until `maybe_exit()` returns `Some` or we get interrupted by a termination
     /// signal.
-    #[inline]
```

Comment:
Intentional? If so, then also maybe we should think about removing `#[inline]` from some of the other `async` methods here.

Reply:
IIRC it was removed because that method used to be very short and then grew, without anyone removing the `#[inline]` declaration.

It is in theory academical at this point, since the method is `pub(crate)` now.

Reply:
While inlining, loop unraveling, and caching are powerful optimization techniques, they should be guided by evidence IMO.  This was done with your deploy caching @marc-casperlabs .